Python/optimizer_bytecodes.c

```diff
@@ -558,8 +558,27 @@ dummy_func(void) {
         self = sym_new_not_null(ctx);
     }
 
-    op(_CHECK_FUNCTION_EXACT_ARGS, (callable, self_or_null, unused[oparg] -- callable, self_or_null, unused[oparg])) {
+    op(_CHECK_FUNCTION_VERSION, (func_version/2, callable, self_or_null, unused[oparg] -- callable, self_or_null, unused[oparg])) {
+        (void)self_or_null;
+        if (sym_is_const(callable) && sym_matches_type(callable, &PyFunction_Type)) {
+            assert(PyFunction_Check(sym_get_const(callable)));
+            REPLACE_OP(this_instr, _CHECK_FUNCTION_VERSION_INLINE, func_version, (uintptr_t)sym_get_const(callable));
```

Comment:
I'm not sure this is safe, since `func_version` is 32 bits but the oparg is only 16.

Perhaps we should do the "two operands per uop" thing and fix the uops that abuse `oparg` like this first?

Reply:
Oops. I forgot `/2` means 32 bits not 16. Yeah you're right.

Reply:
I decided to just check for overflow. We can remove this in the future when we support 2 operands. This is blocking partial evaluation function inlining. So I could really like to get it in.

Reply:
Nevermind, will use this PR as a testing for 2 operand uops.